src/client/mockttp-admin-request-builder.ts

```diff
@@ -303,6 +303,44 @@ export class MockttpAdminRequestBuilder {
                         body
                     }
                 }
+            }`,
+            // TODO: This is just a copy of the previous client-error section. Needs to be finished.
+            'handle-error': gql`subscription OnError {
+                failedRequest {
+                    errorCode
```

Comment:
To make this work, you'll need to:

* Expose it in the API in https://github.com/httptoolkit/mockttp/blob/main/src/admin/mockttp-admin-model.ts (you can just copy the other events there)
* Define it in the schema [here](https://github.com/httptoolkit/mockttp/blob/6941288d9d2c86da5d28b823dc3207b541738a66/src/admin/mockttp-schema.ts#L21) - I'd suggest creating a new `Error` type with `message` and `stack` string fields.
* Adding a case to query for it in the client in [here](https://github.com/httptoolkit/mockttp/blob/6941288d9d2c86da5d28b823dc3207b541738a66/src/client/mockttp-admin-request-builder.ts#L182) and some logic in [here](https://github.com/httptoolkit/mockttp/blob/6941288d9d2c86da5d28b823dc3207b541738a66/src/client/mockttp-admin-request-builder.ts#L312-L327) to turn the `{ message, stack }` data into a proper `Error` object.

I think that should be it, and then the identical API will work in browser & remote clients automatically too.